### PR TITLE
feat(chat): inform user about Name and Version field restrictions in Deployed state (Issue #2693)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Need to inform user why Name and Version fields are not available for edit when code app in Deployed state

Issues:

- Issue #2693 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2713.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — COMMENTED** [review]
Need to add tooltips for disabled name and version

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2713.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeAppView.tsx [hunk 1]
```diff
@@ -199,6 +199,7 @@ export const CodeAppView: FC<ViewProps> = ({
           id="name"
           error={errors.name?.message}
           disabled={isAppDeployed}
+          tooltip={t('Undeploy application to edit name') || ''}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeAppView.tsx]: Tooltip should be displayed only if disabled

File: apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeAppView.tsx [hunk 2]
```diff
@@ -211,6 +212,7 @@ export const CodeAppView: FC<ViewProps> = ({
           name="version"
           disabled={isAppDeployed}
           rules={validators['version']}
+          tooltip={t('Undeploy application to edit version') || ''}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeAppView.tsx]: Tooltip should be displayed only if disabled

File: apps/chat/src/components/Common/Forms/TooltipWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  tooltip?: string;
+}
+
+export function TooltipWrapper<T extends object, R>(
+  Component: ComponentType<T>,
+) {
+  const EnhancedComponent = forwardRef<R, TooltipWrapperProps & T>(
+    ({ tooltip, ...props }, ref) => (
+      <div>
+        {tooltip && (
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/Forms/TooltipWrapper.tsx]: this logic already exists into `Tooltip` component
image

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2713.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**